packages/run-it/src/components/RequestForm/formUtils.tsx

```diff
@@ -296,3 +297,25 @@ export const updateNullableProp = (
   }
   return updatedState
 }
+
+/**
+ * Returns an error message if the body is not JSON or application/x-www-form-urlencoded
```

Comment:
Yeah, the form encoded validation isn't done yet. I think I may have just come up with a regex pattern to use to check it for valid syntax if it appears the body string _is_ form encoded. I meant to make this PR a draft (I have now) because it wasn't quite done. I do appreciate you looking at it, though.